sklearn/metrics/pairwise.py

```diff
@@ -1227,6 +1228,15 @@ def linear_kernel(X, Y=None, dense_output=True):
     return safe_sparse_dot(X, Y.T, dense_output=dense_output)
 
 
+@validate_params(
+    {
+        "X": ["array-like", "sparse matrix"],
+        "Y": ["array-like", "sparse matrix", None],
+        "degree": [Interval(Integral, 1, None, closed="left")],
+        "gamma": [Interval(Real, None, None, closed="neither"), None],
+        "coef0": [Interval(Real, None, None, closed="neither")],
+    }
+)
```

Comment:
In line 1244, I would remove the `:math:` environment because we want to use the parameter naming and it will not render properly. Let's also change `^` by `**`